content/about/notes/1.0.9/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Istio 1.0.9
+publishdate: 2019-06-28
+icon: notes
+release: 1.0.9
+---
+
+This release includes several bug fixes and improvements to robustness.  This release note describes what's different between Istio 1.0.8 and Istio 1.0.9.
```

Comment:
Here and in the other, please remove "several". Probably also remove "improvements" in this case, it's just bug fixes.